cmd/runtimecfg/node-ip.go

```diff
@@ -142,7 +166,7 @@ func checkAddressUsable(chosen []net.IP) (err error) {
 	// If using IPv6, verify that the choosen address isn't tentative
 	// i.e. we can actually bind to it
 	if len(chosen) > 0 && net.IPv6len == len(chosen[0]) {
-		_, err = net.Listen("tcp", "["+chosen[0].String()+"]:")
+		_, err = net.Listen("tcp", "["+chosen[0].String()+"]")
```

Comment:
You may need to keep the : in the address string. I'm not positive, but the docs suggest that's the correct syntax.